auto_cpufreq/core.py

```diff
@@ -413,7 +449,8 @@ def deploy_daemon_performance():
 
     # output warning if gnome power profile is running
     gnome_power_detect_install()
-    gnome_power_svc_disable_performance()
+    #"gnome_power_svc_disable_performance" is not defined
+    # gnome_power_svc_disable_performance()
```

Comment:
Why was `gnome_power_svc_disable_performance()` removed here? 

and why was `#"gnome_power_svc_disable_performance" is not defined` added?

Reply:
Oh, wow this is a great catch. I now also see this function doesn't exist. 

Let's leave it as `#"gnome_power_svc_disable_performance" is not defined`

Reply:
thanks, leaving it commented and resolving the conversation